packages/components/src/card/types.ts

```diff
@@ -71,11 +81,17 @@ type MarginalSubComponentProps = BaseSubComponentProps & {
 	 * @default false
 	 */
 	isBorderless?: boolean;
-	className?: string;
 };
 
 export type HeaderProps = MarginalSubComponentProps;
 
 export type FooterProps = MarginalSubComponentProps & {
 	justify: CSSProperties[ 'justifyContent' ];
 };
+
+export type MediaProps = {
```

Comment:
I know this was [discussed in the previous PR](https://github.com/WordPress/gutenberg/pull/32561#discussion_r649100240), but using `React.ReactNode` directly in `packages/components/src/card/card-media/hook.js` would have been a bit messy, since it would require adding a `React` import at the top of the file, which would trigger 2 eslint errors:

- disabling the rule that disallows importing directly from `react`
- disabling the rule that requires using the `import type` statement (since that statement doesn't work in JS files).

Reply:
> disabling the rule that disallows importing directly from react
> disabling the rule that requires using the import type statement (since that statement doesn't work in JS files).

Neither of these should be true of a JS file that imports a type from `react` as it would use `import('react').ReactNode` which will not trigger either of the conditions you mentioned.

I don't mind leaving this prop type but if you wanted to remove it it should be possible without triggering those rules.

Reply:
Forgive my late-week brain, I completely forgot about the possibility of importing types directly in TSDoc comments.

Done in https://github.com/WordPress/gutenberg/pull/32566/commits/26c84e676e8bcfae1cc829d4bcbeebaa7c6e63dd